### DISCUSSION TITLE
User registration custom fields in admin email

### DISCUSSION THREAD

**Discussion by PixedBo** [joomla/joomla-cms#37304] [2. Cold Features]
Hello everyone!
Just like the contact form's custom fields are sent in admin email it would be great to have user's custom fields filled in registration form to be sent in admin email too.

This is my case: i have a very simple website where users register and fill all kind of custom fields data. Then ad admin can enable the users if some conditions are met. Now the workflow is this one:

- User register and fill all custom fields
- Admin receive the email with the link to activate the user
- Admin needs to log-in in admin panel, search the user, and read the filled custom fields
- Admin then need to return back to the email and click on the activation link (he can't activate the user from backend because in this this way the "activated account" email is not sent to the user)

If i'm able to add the custom fields in the admin (or user) email the process is way more fast and user friendly.

Unfortunately i'm not that skilld in joomla's programming so can't do it myself but this could be a good addition to joomla's core system.

Thanks for your time and kind feebdack.

Gianluca

**Comment by bahamadigital** [comment]
I agree this would be incredibly beneficial!
